src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java

```diff
@@ -0,0 +1,33 @@
+package com.intuit.springwebclient.retryHandler;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+public class RetryHandlerFactoryTest {
+  
+  private RetryHandlerFactory retryHandlerFactory;
+
+  @BeforeEach
+  public void setUp() {
+    retryHandlerFactory = new RetryHandlerFactory();
+  }
+
+  @Test
+  public void testAddAndGetHandler() {
+    RetryHandler retryHandler = new DefaultRetryHandler();
```

Comment:
DefaultRetryHandler is not imported in this class

Reply:
Updated the PR, handled this.